app/fissile.go

```diff
@@ -772,7 +773,7 @@ func compareHashes(v1Hash, v2Hash keyHash) *HashDiffs {
 
 // GenerateKube will create a set of configuration files suitable for deployment
 // on Kubernetes
-func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registry, organization, fissileVersion string, defaultFiles []string, useMemoryLimits bool, opinions *model.Opinions) error {
+func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registry, organization, fissileVersion string, defaultFiles []string, useMemoryLimits bool, createHelmChart bool, chartFilename string, opinions *model.Opinions) error {
```

Comment:
I think keeping kube configs a little while longer is not a bad idea.
Helm might not be available everywhere yet.